client/consensus/babe/src/aux_schema.rs

```diff
@@ -65,6 +68,9 @@ pub fn load_epoch_changes<Block: BlockT, B: AuxStore>(
 		Some(1) =>
 			load_decode::<_, EpochChangesFor<Block, EpochV0>>(backend, BABE_EPOCH_CHANGES_KEY)?
 				.map(|v1| v1.map(|_, _, epoch| epoch.migrate(config))),
+		Some(2) =>
+			load_decode::<_, EpochChangesForV1<Block, Epoch>>(backend, BABE_EPOCH_CHANGES_KEY)?
```

Comment:
There's some confusion to versions here. `V1` means version of the epoch definition, rather than epoch changes. I'll rename the type to indicate the version of epoch changes instead.